package.json

```diff
@@ -47,7 +51,7 @@
     "svelte-loader": "3.1.4"
   },
   "peerDependencies": {
-    "typescript": "4.9.4"
+    "typescript": "^4.7.0"
```

Comment:
Why are we reverting to a previous version here?

Reply:
It's a peer dependency, and we want to be as permissive as possible here